### ISSUE TITLE
Feature Request: keeping filters active when jumping between boards

### ISSUE THREAD

**Issue by triple-doble** [wekan/wekan#1751]
Hello.

I use filters all the time, mostly to have only visible my user´s cards and not the other users ones.
But, when jumping to other boards, I need to establish the filter again. I think it would be nice to have an option to keeping the filters active.

**Comment by TiibCD** [comment]
I agree this would be nice I had the same thought. I use custom fields filters all the time otherwise some boards are rapidly full with cards

**Comment by d3dbit** [comment]
Great idea!

**Comment by TiibCD** [comment]
Does anyone plan to work on this in the future ?

**Comment by xet7** [comment]
If someone is interested in this, filter related PRs in Wekan are here:
https://github.com/wekan/wekan/pulls?q=is%3Apr+filter+is%3Aclosed

Some filter code is here:
https://github.com/wekan/wekan/blob/edge/client/lib/filter.js

Filter options would be saved to boards for user, in schema of board:
https://github.com/wekan/wekan/blob/edge/models/boards.js

And migrations for new filter database field would be added to here:
https://github.com/wekan/wekan/blob/edge/server/migrations.js

PR could be added to edge branch.
